docs/UserGuide.md

```diff
@@ -3,271 +3,560 @@ layout: page
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-![without policy URL](images/without_policy_URL.png)
-![with policy URL](images/with_policy_URL.png)
+  ![delete charlotte](images/delete-charlotte.png)
 
-Format: `policy INDEX`
+[Return to Table of Contents](#table-of-contents)
+<br><br>
 
-* Selects the client at the specified `INDEX`.
-* The index refers to the index number shown in the displayed client list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* Policy ID cannot contain '>' character!
-* Note that your URLs with should not contain '>' characters either! They are not valid URLs by the Internet's definition.
 
-Examples:
-* `list` followed by `policy 2` displays the policies associated with the 2nd person in the address book.
-* `find Betsy` followed by `policy 1` displays the policies associated with the 1st person in the results of the `find` command.
-
-### Sorting the list of clients : `sort`
+### `sort`: Sort list of clients
 
-Sorts the list of clients in ClientBook.
+**Purpose**: Sorts the current list of clients in ClientBook.
 
-Format: `sort ATTRIBUTE DIRECTION`
+**Format**: `sort -ATTRIBUTE -DIRECTION`
 
 * Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
-* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy.
-* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order.
+* The specified `ATTRIBUTE` can be `-n` to sort by name alphabetically or `-i` to sort by number of insurance policies, but not both.
+* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order, but not both.
+
+**Examples**:
+* Sort the current list of clients by name in **descending** number of insurance policies.
+    * `sort -i -des`
+
+      ![sort](images/sort-policy-des.png)
 
-Examples:
-* `sort -n -asc` sorts the list of clients by name in ascending order.
+* Sort the current list of clients by name in **descending** alphabetical order.
+    * `sort -n -des`
 
-### Exiting the program : `exit`
+      ![sort](images/sort-des.png)
 
-Exits the program.
+[Return to Table of Contents](#table-of-contents)
+<br><br>
 
-Format: `exit`
 
-### Locking ClientBook : `lock`
+### `lock`: Lock ClientBook with a user-selected password 
 
-Locks ClientBook.
+**Purpose**: Locks ClientBook with a user-selected password.
 
-Format: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
+**Format**: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
 
 * Verifies the current password before locking ClientBook with the new password.
 * The `CURRENT_PASSWORD` field can be omitted if ClientBook is not yet locked.
 * When `CURRENT_PASSWORD` and `NEW_PASSWORD` fields are both omitted, ClientBook will attempt to lock itself using the last used password that is safely stored on your device.
```

Comment:
A little nitpicking, but instead of leaving it as the default case, it might be better to specify each specific case, like

* The `CURRENT_PASSWORD` field can be omitted if ClientBook is not yet locked.
* When `CURRENT_PASSWORD` and `NEW_PASSWORD` are both specified, ClientBook verifies the current password before locking ClientBook with the new password.  <- specifying this case instead of leaving it as default case
* When `CURRENT_PASSWORD` and `NEW_PASSWORD` fields are both omitted, ClientBook will attempt to lock itself using the last used password that is safely stored on your device.